backend/scripts/application_stop.sh

```
@@ -1,58 +1,51 @@
     rm -f app/coffee-shout.pid
 else
-    echo "   ℹ️ PID 파일이 없습니다. 애플리케이션이 실행 중이 아닐 수 있습니다"
+    echo "   ℹ️  PID 파일이 없습니다"
 fi
 
 # 포트 8080 사용 프로세스 강제 종료 (혹시 모를 좀비 프로세스)
 JAVA_PROCESS=$(lsof -ti:8080 2>/dev/null || true)
 if [ ! -z "$JAVA_PROCESS" ]; then
-    echo "   🔫 포트 8080을 사용하는 프로세스 강제 종료 (PID: $JAVA_PROCESS)"
+    echo "   🔫 포트 8080을 사용하는 좀비 프로세스 강제 종료 (PID: $JAVA_PROCESS)"
     kill -9 $JAVA_PROCESS 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)